capsules/detector_text_openvino/README.md

```diff
@@ -0,0 +1,11 @@
+# About the Capsule
+## Usage
+This capsule is for detecting text in video. 
+
+## Model
+###  Model File Origin
+The pretrained model was downloaded from the [OpenCV Open Model Zoo](https://github.com/opencv/open_model_zoo). 
+For more information on this specific model, see its 
+[model page](https://docs.openvinotoolkit.org/2019_R1/_text_detection_0002_description_text_detection_0002.html).
```

Comment:
This information is old. I recommend explaining a bit more here. 

Something explaining the architecture involving a detector, recognizer-encoder, and recognizer-decoder. Here are the relevant links to include:

https://docs.openvinotoolkit.org/2020.3/_models_intel_text_spotting_0002_detector_description_text_spotting_0002_detector.html
https://docs.openvinotoolkit.org/2020.3/_models_intel_text_spotting_0002_recognizer_encoder_description_text_spotting_0002_recognizer_encoder.html
https://docs.openvinotoolkit.org/2020.3/_models_intel_text_spotting_0002_recognizer_decoder_description_text_spotting_0002_recognizer_decoder.html